### PR TITLE
Improved LICENSE.txt

### PR DESCRIPTION
**Pull request by yashmit178**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Design and Layout
 
 The design, layout, and look-and-feel of dojo-website content - including
-all CSS, images, and icons, except as noted above - are copyright JS
+all CSS, images, and icons, except as noted above - are copyright of JS
 Foundation and are not licensed for use. Designs and CSS from
-dojo-website may not be used on any site, personal or commercial,
+dojo-website may not be used on any site for personal or commercial use 
 without prior written consent from the JS Foundation.
```
Comment (dylans) [inline LICENSE.txt:7]:
```suggestion
without prior written consent from the OpenJS Foundation.
```

**Review by dylans — CHANGES_REQUESTED** [review]
Thanks. These changes made me realize we need a few other changes

* JS Foundation -> OpenJS Foundation in several places
* 2017 -> 2020

Let me know if you'd like to make these changes. If not I'll do so in a separate PR.

**Comment by yashmit178** [comment]
Will be keen to do the same, thanks for the opportunity Dylan

**Review by dylans — APPROVED** [review]

**Comment by dylans** [comment]
Thanks @yashmit178!

**Comment by yashmit178** [comment]
i should thank you as you gave me an opportunity to build my proile and i
felt so motivated after helping you.

**Pull request merged**